tests/test-config.json.example

```diff
@@ -3,9 +3,9 @@
   "api_key": "demo",
   "service_key":"demo",
   "teamserver_url": "http://127.0.0.1:19080",
-  "org_uuid": "de7dd2a9-7643-46fb-b9d8-9c38aeffed7e",
+  "org_uuid": "4b7bc7b8-c59d-43ee-8b75-9c5172535f7b",
```

Comment:
this is just a template no changes are required in this file

Reply:
Restored to the original value